jsoc/index.md

```diff
@@ -26,6 +26,13 @@ Those participating in the program can check out [the getting started page](/lea
 Are you interested in Julia but not super familiar with the language?
 Head over to [JuliaAcademy](https://juliaacademy.com) to take some free online Julia classes.
 
+## Google Season of Docs (GSoD)
+
+The Julia organization is applying to participant in [Google's Season of Docs](https://developers.google.com/season-of-docs).
```

Comment:
applying to participate ?

Reply:
Yeah, we are not in yet, but we need to have out projects list publicly available

Reply:
oh sorry I meant `participant <> participate`